### ISSUE TITLE
Podman failing to start on MacOS

### ISSUE THREAD

**Issue by lordbeerus0505** [containers/podman#20428]
### Issue Description

I installed Podman with `brew install podman` and then tried to run init with 
`podman machine init`
Followed this up with 
`podman machine start` but that left me with 
```
$ podman machine start
Starting machine "podman-machine-default"
Waiting for VM ...
Error: machine did not transition into running state
```
It has been doing this for quite some time but used to work fine about 1 month ago when I last tried. I have tried reinstalling it after clearing all the files and yet it leads to the same result.
Is there some other non brew mechanism to install (say from a tar.gz)?

### Steps to reproduce the issue

Steps to reproduce the issue
1. brew install podman
2. podman machine init
3. podman machine start

### Describe the results you received

Waiting for VM ...
Error: machine did not transition into running state

### Describe the results you expected

I expect it to start and show file system mounts.

### podman info output

```yaml
podman version 4.7.1
```

### Podman in a container

No

### Privileged Or Rootless

None

### Upstream Latest Release

Yes

### Additional environment details

Additional environment details

### Additional information

This happens on my Macbook Pro (M1 Pro) but works fine on my Macbook Air (M1).

**Comment by Luap99** [comment]
Looks like a duplicate of https://github.com/containers/podman/issues/20081

**Comment by lclc** [comment]
I run into the same issue.

- Fresh MacBook M3 Pro
- Fresh installed Podman with brew.
- podman version 4.8.3

```
lucas@MBPvonLucas2023 Downloads % podman machine init
Extracting compressed file: podman-machine-default_fedora-coreos-39.20231204.2.1-qemu.aarch64.qcow2: done
Image resized.
Machine init complete
To start your machine run:

	podman machine start

lucas@MBPvonLucas2023 Downloads % podman machine start --log-level=debug
INFO[0000] podman filtering at log level debug
DEBU[0000] Using Podman machine with `qemu` virtualization provider
Starting machine "podman-machine-default"
DEBU[0000] {true 1500 map[forward-dest:[/run/user/501/podman/podman.sock] forward-identity:[/Users/lucas/.ssh/podman-machine-default] forward-sock:[/Users/lucas/.local/share/containers/podman/machine/qemu/podman.sock] forward-user:[core]] [] map[listen-qemu:unix:///var/folders/vc/70pl1w4n3t73_g925yb6yclh0000gn/T/podman/qmp_podman-machine-default.sock] /var/folders/vc/70pl1w4n3t73_g925yb6yclh0000gn/T/podman/podman-machine-default_proxy.pid 49380}
DEBU[0000] qemu cmd: [/opt/homebrew/bin/qemu-system-aarch64 -accel hvf -accel tcg -cpu host -M virt,highmem=on -drive file=/opt/homebrew/share/qemu/edk2-aarch64-code.fd,if=pflash,format=raw,readonly=on -drive file=/Users/lucas/.local/share/containers/podman/machine/qemu/podman-machine-default_ovmf_vars.fd,if=pflash,format=raw -m 2048 -smp 6 -fw_cfg name=opt/com.coreos/config,file=/Users/lucas/.config/containers/podman/machine/qemu/podman-machine-default.ign -qmp unix:/var/folders/vc/70pl1w4n3t73_g925yb6yclh0000gn/T/podman/qmp_podman-machine-default.sock,server=on,wait=off -netdev socket,id=vlan,fd=3 -device virtio-net-pci,netdev=vlan,mac=5a:94:ef:e4:0c:ee -device virtio-serial -chardev socket,path=/var/folders/vc/70pl1w4n3t73_g925yb6yclh0000gn/T/podman/podman-machine-default_ready.sock,server=on,wait=off,id=apodman-machine-default_ready -device virtserialport,chardev=apodman-machine-default_ready,name=org.fedoraproject.port.0 -pidfile /var/folders/vc/70pl1w4n3t73_g925yb6yclh0000gn/T/podman/podman-machine-default_vm.pid -virtfs local,path=/Users,mount_tag=vol0,security_model=none -virtfs local,path=/private,mount_tag=vol1,security_model=none -virtfs local,path=/var/folders,mount_tag=vol2,security_model=none -drive if=virtio,file=/Users/lucas/.local/share/containers/podman/machine/qemu/podman-machine-default_fedora-coreos-39.20231204.2.1-qemu.aarch64.qcow2]
Waiting for VM ...
```

Then it just hangs at "Waiting for VM ..."


The output on QEMU:
![image](https://github.com/containers/podman/assets/1846033/9b63cc27-b44f-46df-bbca-510462ac3e4f)

**Comment by ashley-cui** [comment]
Probably solved with 5.0, please re-open if this is not the case
